adafruit_rgb_display/ssd1331.py

```diff
@@ -62,31 +75,42 @@ class SSD1331(DisplaySPI):
     _INIT = (
         (_DISPLAYOFF, b''),
         (_LOCK, b'\x0b'),
-        (_SETREMAP, b'\x72'), # RGB Color
+        (_SETREMAP, b'\x72'),  # RGB Color
         (_STARTLINE, b'\x00'),
         (_DISPLAYOFFSET, b'\x00'),
         (_NORMALDISPLAY, b''),
-#        (_FILL, b'\x01'),
+        # (_FILL, b'\x01'),
 
-#	(_PHASEPERIOD, b'\x31'),
-#	(_SETMULTIPLEX, b'\x3f'),
-#	(_SETMASTER, b'\x8e'),
-#	(_POWERMODE,b'\x0b'),
-#	(_PRECHARGE, b'\x31'), #;//0x1F - 0x31
-#	(_CLOCKDIV, b'\xf0'),
-#	(_VCOMH, b'\x3e'), #;//0x3E - 0x3F
-#	(_MASTERCURRENT, b'\x06'), # ;//0x06 - 0x0F
-#	(_PRECHARGEA, b'\x64'),
-#	(_PRECHARGEB, b'\x78'),
-#	(_PRECHARGEC, b'\x64'),
-#	(_PRECHARGELEVEL, b'\x3a'), # 0x3A - 0x00
-#	(_CONTRASTA, b'\x91'), #//0xEF - 0x91
-#	(_CONTRASTB, b'\x50'), #;//0x11 - 0x50
-#	(_CONTRASTC, b'\x7d'), #;//0x48 - 0x7D
+        (_PHASEPERIOD, b'\x31'),
+        (_SETMULTIPLEX, b'\x3f'),
+        (_SETMASTER, b'\x8e'),
+        (_POWERMODE, b'\x0b'),
+        (_PRECHARGE, b'\x31'),  # ;//0x1F - 0x31
+        (_CLOCKDIV, b'\xf0'),
+        (_VCOMH, b'\x3e'),  # ;//0x3E - 0x3F
+        (_MASTERCURRENT, b'\x0c'),  # ;//0x06 - 0x0F
+        (_PRECHARGEA, b'\x64'),
+        (_PRECHARGEB, b'\x78'),
+        (_PRECHARGEC, b'\x64'),
+        (_PRECHARGELEVEL, b'\x3a'),  # 0x3A - 0x00
+        (_CONTRASTA, b'\x91'),  # //0xEF - 0x91
+        (_CONTRASTB, b'\x50'),  # ;//0x11 - 0x50
+        (_CONTRASTC, b'\x7d'),  # ;//0x48 - 0x7D
         (_DISPLAYON, b''),
     )
     _ENCODE_PIXEL = ">H"
     _ENCODE_POS = ">BB"
 
+    # pylint: disable-msg=useless-super-delegation, too-many-arguments
     def __init__(self, spi, dc, cs, rst=None, width=96, height=64):
         super().__init__(spi, dc, cs, rst, width, height)
+
+    # pylint: disable-msg=no-member
```

Comment:
Please add a comment why each of these lint exemptions are needed.

Reply:
We need the `__init__` here to provide default values, so it's not really useless.

Reply:
Ok, thats what I was wondering.